src/mainMenuScene.js

```diff
@@ -26,13 +26,7 @@ const menuActionList = [
 ];
 
 class MainMenuScene {
-    inputManager;
```

Comment:
Is there a reason you'd like this to be changed? 

The code is using standard ECMA class [field declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#field_declarations) syntax, which is additionally used throughout the rest of the code base (along with other things like private members.